app/views/onboard/check.njk

```diff
@@ -0,0 +1,124 @@
+{% extends "_layout.njk" %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    href: "/onboard/step4"
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <span class="govuk-caption-xl">Manage permissions</span>
+      <h1 class="govuk-heading-xl">Check permissions</h1>
+
+      <form novalidate action="/">
+
+        {% set permissionsOneHtml %}
+          <p>University of Brighton can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+
+          <p>The Millais Alliance can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+        {% endset %}
+
+        {% set permissionsTwoHtml %}
+          <p>University of Sussex can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+
+          <p>The Millais Alliance can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+        {% endset %}
+
+        {% set permissionsThreeHtml %}
+          <p>University of Chichester can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+
+          <p>The Millais Alliance can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+        {% endset %}
+
+        {{ govukSummaryList({
```

Comment:
The summary list seems a bit unwieldy here. Would it work better is the content was able to stretch the full width of the page?

<img width="680" alt="Screenshot 2020-04-21 at 14 41 59" src="https://user-images.githubusercontent.com/813383/79872953-56c5bf80-83de-11ea-9dbe-9536d8ced0cb.png">

Reply:
Yeah, it is, will try it out!